mod/metrics/set/type_plus_right/metric/formula/wiki_rating.rb

```diff
@@ -18,7 +18,23 @@
   # table with Metrics on left and Weight inputs on right
   def rating_editor_table
     table rating_editor_table_content, class: "wikiRating-editor",
-                                       header: %w[Metric Weight]
+                                       header: ["Metric", checkbox_equalizer]
+  end
+
+  def checkbox_equalizer
```

Comment:
perhaps "weight_heading" would be a clearer name? (with a comment that it includes the equalizer checkbox?